Bottom: 2a6fd13d7624d6ea5294323f3973a03d234ad895
Top:    f2a595a328c78959b05029f7f3da09a86dfc5d51
Author: Pavel Roskin <proski@gnu.org>
Date:   2014-12-23 22:10:14 -0500

Refresh of linux33

---

diff --git a/farm.conf b/farm.conf
index 0180ae2..d652dd9 100644
--- a/farm.conf
+++ b/farm.conf
@@ -17,7 +17,7 @@ linuxgit=$HOME/src/linux
 tags="v2.6.13 v2.6.14 v2.6.15 v2.6.16 v2.6.17 v2.6.18 v2.6.19 v2.6.20
 v2.6.21 v2.6.22 v2.6.23 v2.6.24 v2.6.25 v2.6.26 v2.6.27 v2.6.28 v2.6.29
 v2.6.30 v2.6.31 v2.6.32 v2.6.33 v2.6.34 v2.6.35 v2.6.36 v2.6.37 v2.6.38
-v2.6.39 v3.0 v3.1 v3.2 v3.3.8 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11
+v2.6.39 v3.0 v3.1 v3.2 v3.3 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11
 v3.12 v3.13 v3.14 v3.15 v3.16 v3.17 v3.18 linux-next/master"
 
 # Kernel configuration variables to enable
diff --git a/kernel-patches/Makefile.am b/kernel-patches/Makefile.am
index 9abc3c0..c725643 100644
--- a/kernel-patches/Makefile.am
+++ b/kernel-patches/Makefile.am
@@ -2,4 +2,5 @@ patchdir = $(pkgdatadir)/kernel-patches
 
 dist_patch_DATA = \
 	0001-makefile-config-2.6.14.patch \
-	0002-makefile-config-2.6.18.patch
+	0002-makefile-config-2.6.18.patch \
+	0003-cmpxchg-3.3.patch
